File: test/data/modifier_audit/async_functions.swift
func syncHelper() -> Int {
    return 42
}

func fetchData(url: String) async -> String {
    return await URLSession.shared.data(from: url)
}

static func staticHelper() -> Void {
    print("static")
}

mutating func update() {
    self.count += 1
}

public func publicHelper() -> Bool {
    return true
}

File: test/data/modifier_audit/async_functions.dart
Future<String> fetchData(String url) async {
  return await http.get(url);
}

String syncHelper() {
  return 'hello';
}

Stream<int> generateNumbers() async* {
  for (var i = 0; i < 10; i++) {
    yield i;
  }
}

class Service {
  static void staticHelper() {
    print('static');
  }

  Future<void> asyncMethod() async {
    await Future.delayed(Duration(seconds: 1));
  }
}
